demo/admin/generate-block-types.ts

```diff
@@ -73,19 +73,19 @@ function writeFieldType(field: BlockMetaField, blockNamePostfix: string) {
         content += "{\n";
         field.object.fields.forEach((f) => {
             content += f.name;
-            if (f.nullable) content += "?";
             content += ": ";
             writeFieldType(f, blockNamePostfix);
+            if (f.nullable) content += " | null";
```

Comment:
Demo should use `@comet/cli` to generate the types

Reply:
yes, I noticed that too :) but that is out of scope for this pr

Reply:
https://github.com/vivid-planet/comet/pull/1066